controllers/argocd/rolebinding.go

```diff
@@ -340,6 +340,11 @@ func newRoleBindingWithNameForApplicationSourceNamespaces(namespace string, cr *
 
 func (r *ReconcileArgoCD) reconcileClusterRoleBinding(name string, role *v1.ClusterRole, cr *argoproj.ArgoCD) error {
 
+	if cr.Spec.DefaultClusterScopedRoleCreationDisabled {
```

Comment:
If `DefaultClusterScopedRoleCreationDisabled` is set to  true, should we delete the cluster roles bindings if they already exist?